libsolidity/codegen/ir/IRGenerator.cpp

```diff
@@ -130,6 +130,7 @@ string IRGenerator::generate(
 	};
 
 	Whiskers t(R"(
+		<sourceLocationComment>
```

Comment:
The object node does not have debug data associated, so could you move this to the beginning of the code (both creation and deployed)?